frontend/javascript/components/makerequest/user-create-account.vue

```diff
@@ -0,0 +1,180 @@
+<template>
```

Comment:
This view is a little cluttered right now, with lots of text visible at once. The sign in link is hard to spot. Maybe it makes sense to first ask the user whether to login or register, then show the register form, possibly split into sub-steps?